app/routes/qr+/$qrId_.contact-owner.tsx

```diff
@@ -34,7 +34,7 @@ export const action = async ({ request, params }: ActionArgs) => {
 
   const owner = await getUserByID(qr.userId);
   if (!owner) return new Response("Something went wrong", { status: 500 });
-  const asset = await getAsset({ organizationId, id: qr.assetId });
+  const asset = await getAsset({ userId, id: qr.assetId });
```

Comment:
Hey man. This is still not correct, i think i might have not explained it correctly. You need to find the asset based on the QR ID because this flow triggers only if the asset doesn't belong to the user performing the action. The 
So you need to search for the asset based on its `qrId` which you can get via the params.
```ts
const qrId = getRequiredParam(params, "qrId");
```

Reply:
I took a few minutes to make the changes because I wanted to merge. Thanks for PR, lets continue with the other models.